eslint.config.mjs

```diff
@@ -0,0 +1,47 @@
+import base, { createConfig } from '@metamask/eslint-config';
+import jest from '@metamask/eslint-config-jest';
+import nodejs from '@metamask/eslint-config-nodejs';
+import typescript from '@metamask/eslint-config-typescript';
+
+const config = createConfig([
+  {
+    ignores: ['!.eslintrc.js', '!.prettierrc.js', 'dist/', 'docs/', '.yarn/'],
```

Comment:
As we removed `.eslintrc.js` and renamed `.prettierrc.js`, should this list also be changed?

Reply:
Good catch! It seems like these are no longer ignored by default in ESLint 9, so I will remove them.